src/homepageExperience/components/steps/ExecuteAggregateQuery.tsx

```diff
@@ -1,21 +1,45 @@
 import React from 'react'
 import CodeSnippet from 'src/shared/components/CodeSnippet'
 
-export const ExecuteAggregateQuery = () => {
-  const codeSnippet = `query_api = client.query_api()
+import {SafeBlankLink} from 'src/utils/SafeBlankLink'
+
+const fromBucketSnippet = `from(bucket: “my-bucket”)
```

Comment:
just moving this variable declaration to outside the function component. Probably immaterial, but feels dirty to re-create an identical variable every render when we don't need to